contracts/casino/include/casino/casino.hpp

```diff
@@ -45,12 +47,34 @@ struct [[eosio::table("gamestate"), eosio::contract("casino")]] game_state_row {
 
 using game_state_table = eosio::multi_index<"gamestate"_n, game_state_row>;
 
+struct [[eosio::table("global"), eosio::contract("casino")]] global_state {
+    asset game_active_sessions_sum;
+    asset game_profits_sum;
+    time_point last_claim_time;
```

Comment:
mb better name is last_withdraw_time, because of casino do withdraw operation(not claim)